docs/DeveloperGuide.md

```diff
@@ -2,8 +2,7 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-{:toc}
+* Table of Contents{:toc}
```

Comment:
Hmm is this change here intended? 🤔

Reply:
No man trying to solve merge conflicts w copy and paste led to a disaster